src/js/form-builder.js

```diff
@@ -730,9 +731,43 @@
 
       advFields.push(textAttribute('maxlength', values));
 
-      return advFields.join('');
-    };
-
+      //+ gimigliano
+      if(opts.typeUserAttrs[values.type]) {}

```

Comment:
Right but i meant that the conditional starts and ends on L735 because it has `{}`. Once thats fixed we can merge.